### PR TITLE
Issue #1

### PR DESCRIPTION
**Pull request by iConsu**
Create endpoint for api/clean and delete all the deliverers

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -49,6 +49,15 @@ router.post('/api/register', async ctx => {
   }
 })
 
+//issue 1
```
Comment (alayor) [inline index.js]: Can you remove this comment?

File: index.js [hunk 2]
```diff
@@ -49,6 +49,15 @@ router.post('/api/register', async ctx => {
...
+router.post('/api/clean', async ctx => {
+  deliverers = []
+  ctx.body = {
+    success: true,
+  }
+})
+//End issue 1
```
Comment (alayor) [inline index.js]: Can you also remove this comment?

File: index.js [hunk 3]
```diff
@@ -49,6 +49,15 @@ router.post('/api/register', async ctx => {
   }
 })
 
+//issue 1
+router.post('/api/clean', async ctx => {
+  deliverers = []
```
Comment (alayor) [inline index.js]: I think it's better to do this `deliverers.length = 0` given that deliverers is a constant. I was wrong there.

File: index.js (hunk 1 above)
Comment (iConsu) [reply index.js]: Sure! :+1:

File: index.js (hunk 2 above)
Comment (iConsu) [reply index.js]: Sure! :+1:

**Pull request merged**